typesystem/formats.py

```diff
@@ -130,11 +130,12 @@ def validate(self, value: typing.Any) -> datetime.datetime:
         except ValueError:
             raise self.validation_error("invalid")
 
-    # def serialize(self, obj: typing.Any) -> str:
-    #     value = value.isoformat()
-    #     if value.endswith('+00:00'):
-    #         value = value[:-6] + 'Z'
-    #     return value
+    def serialize(self, obj: typing.Any) -> str:
+        value = obj.isoformat()
```

Comment:
Be careful that you may get `obj is None` (e.g. if the field has `allow_null=True`).

May I suggest:

```suggestion
        if obj is None:
            return None
        value = obj.isoformat()
```